app/Enums/DatabaseDriver.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Enums;
+
+enum DatabaseDriver: string
+{
+    case Mariadb = 'mariadb';
+    case Mysql = 'mysql';
+    case Postgresql = 'pgsql';
+    case Sqlite = 'sqlite';
+
+    public function getFriendlyName(): string
+    {
+        return match($this) {
+            self::Mariadb => 'MariaDB',
+            self::Mysql => 'MySQL',
+            self::Postgresql => 'PostgreSQL',
+            self::Sqlite => 'SQLite',
+        };
+    }
+
+    public function getJDBCDriver(): string
+    {
+        return match($this) {
+            self::Mariadb, self::Mysql => 'mysql',
+            self::Postgresql => 'postgresql',
+            self::Sqlite => 'sqlite',
+        };
+    }
+
+    public function isRemote(): bool
+    {
+        return match($this) {
+            self::Mariadb, self::Mysql, self::Postgresql => true,
+            self::Sqlite => false,
+        };
+    }
+
+    public function getDefaultOption(string $key, bool $useEnv = false): mixed
```

Comment:
I like this enum, but the config values definitely should be stored in the config file. The getDefaultOption can just pull using config(). I know this won't work for the port for example, but feel free to just make a getDefaultPort method.

Reply:
I wasn't sure what would be best, so I just tried it out.